packages/shared/friends/sagas.ts

```diff
@@ -708,6 +719,84 @@ export async function getFriendRequests(request: GetFriendRequestsPayload) {
   getUnityInstance().AddFriendRequests(addFriendRequestsPayload)
 }
 
+// New friend request flow
+export async function getFriendRequestsProtocol(request: GetFriendRequestsPayload) {
+  try {
+    // Get friends
+    const friends: FriendsState = getPrivateMessaging(store.getState())
+
+    const realmAdapter = await ensureRealmAdapterPromise()
+    const fetchContentServerWithPrefix = getFetchContentUrlPrefixFromRealmAdapter(realmAdapter)
+
+    const fromFriendRequests = friends.fromFriendRequests.slice(
+      request.receivedSkip,
+      request.receivedSkip + request.receivedLimit
+    )
+    const toFriendRequests = friends.toFriendRequests.slice(request.sentSkip, request.sentSkip + request.sentLimit)
```

Comment:
Can you add comments explaining what these vector operations are doing? Looks like it is to scroll through the requests